sdk/iptables/iptables.go

```diff
@@ -0,0 +1,158 @@
+package iptables
```

Comment:
Since this will only support linux, we could set up alternative os-specific implementations and return an error if used in not-linux.

Here's an example where we have an alternative Windows implementation:
https://github.com/hashicorp/consul/blob/87f6617eecd23a64add1e79eb3cd8dc3da9e649e/agent/config/limits_windows.go
https://github.com/hashicorp/consul/blob/87f6617eecd23a64add1e79eb3cd8dc3da9e649e/agent/config/limits.go

Reply:
Done!